buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -38,7 +38,7 @@
       }
     },
     {
-      "enabled": true,
+      "enabled": false,
```

Comment:
We can't remove this step, it's needed for non-Windows (it always fails on Windows). The right fix is to exit with success immediately if this task is running on Windows, which I've been meaning to do for a while now (I can put that up today).